packages/@aws-cdk/aws-secretsmanager/lib/secret.ts

```diff
@@ -148,6 +155,25 @@ abstract class SecretBase extends Resource implements ISecret {
     return result;
   }
 
+  public grantWrite(grantee: iam.IGrantable): iam.Grant {
+    // @see https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access_identity-based-policies.html
+    const result = iam.Grant.addToPrincipal({
+      grantee,
+      actions: ['secretsmanager:PutSecretValue'],
+      resourceArns: [this.secretArn],
+      scope: this,
+    });
+
+    if (this.encryptionKey) {
+      // @see https://docs.aws.amazon.com/kms/latest/developerguide/services-secrets-manager.html
```

Comment:
`@see` is only for JSDocs, doesn't do anything for inline docs, so feel free to remove it :)

Reply:
That's what I get for blindly copying `grantRead` :-)